src/main/resources/application-example.yaml

```
@@ -60,3 +60,8 @@ spring:
           auth: true
           starttls:
             enable: true
+
+discord:
+  bot:
+    token: your_discord_api_token
+    channel-id: your_discord_channel_id
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)